test/pathSatisfies.js

```diff
@@ -14,8 +14,8 @@ describe('pathSatisfies', function() {
     eq(R.pathSatisfies(isPositive, ['x', 'y'], {x: {z: 42}}), false);
   });
 
-  it('returns false if the path is empty', function() {
-    eq(R.pathSatisfies(isPositive, [], {x: {z: 42}}), false);
+  it('handles empty paths by just comparing data', function() {
+    eq(R.pathSatisfies(R.is(Object), [], {x: {z: 42}}), true);
   });
```

Comment:
I think it would be worthwhile to add one negative test as well.  Perhaps `R.has('y')` with the same object.

Reply:
Sounds good. I added a negative, let me know if that satisfies what you were looking for.